cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml

```diff
@@ -4,14 +4,10 @@ metadata:
    name: csi-rbd
 provisioner: rbd.csi.ceph.com
 parameters:
-    # Comma separated list of Ceph monitors
-    # if using FQDN, make sure csi plugin's dns policy is appropriate.
-    monitors: mon1:port,mon2:port,...
-
-    # if "monitors" parameter is not set, driver to get monitors from same
-    # secret as admin/user credentials. "monValueFromSecret" provides the
-    # key in the secret whose value is the mons
-    #monValueFromSecret: "monitors"
+    # Specify the ID of the cluster. This can be anything but is usually
+    # the UUID of your ceph cluster. It must match the content of the
+    # cluster id in the ceph-csi-config config map.
+    clusterID: MY-CLUSTER-ID
```

Comment:
`clusterNamespace`?

Reply:
Value now defaults to the "rook-ceph" , key is defined by CSI.